integration_tests/src/main.rs

```diff
@@ -24,9 +24,9 @@ async fn main() {
     // NOTE: order matters
```

Comment:
Should we move this file to the `util` crate?

Reply:
Hmm... Because you are thinking of using it from ansible? I would move it to the util crate once it's actually used outside this one, rather than before